pkg/protobuf/app/app.proto

```diff
@@ -43,8 +43,9 @@ message CreateRequest {
     Autoscale autoscale = 5;
 
     string virtual_host = 6;
-    bool internal = 7;
-    string protocol = 8;
+    bool reserve_static_ip = 7;
+    bool internal = 8;
+    string protocol = 9;
```

Comment:
hm, maybe is better to maintain the old order and add `reserve_static_ip` as field `9`